src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -163,7 +163,7 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
     public async showPipelineFiles(): Promise<void> {
         this.filesToCommit.forEach(async (file) => {
             await this.localGitRepoHelper.addContentToFile(file.content, file.absPath);
-            await vscode.window.showTextDocument(vscode.Uri.file(file.absPath));
+            await vscode.window.showTextDocument(vscode.Uri.file(file.absPath), { preview: false });
```

Comment:
what does the preview false do?

Reply:
I also want to know what does it do

Reply:
When we set preview to false, it does not replace the content of current editor window. For example, if we don’t set preview to false and call function showTextDocument in a loop for different then same editor window will be used to show the files (not different editor window for different files)